src/UI/Activity/Queue/QueueLayout.js

```diff
@@ -1,7 +1,7 @@
 var Marionette = require('marionette');
 var Backgrid = require('backgrid');
 var QueueCollection = require('./QueueCollection');
-var SeriesTitleCell = require('../../Cells/MovieTitleCell');
+var MovieTitleCell = require('../../Cells/MovieTitleCell');
```

Comment:
It works fine as written and allows itself to _properly_ use a route/title without hacks like the history cell does. No idea why, but it does.